concepticondata/conceptlists.tsv

```diff
@@ -419,4 +419,5 @@ Wilson-2015-173	Wilson, Fiona M.	2015		173	basic, ranked, norms	Latin	Spanish		W
 Chacon-2022-94	Chacon, Thiago C.	2022		94	basic	English,Portuguese	Northwest Arawakan Languages		Chacon2022		This concept list is taken from a dataset on Northwest Arawakan Languages. The list is based on [Swadesh's list of 100 items](:ref:Swadesh-1955-100).		
 Lionnet-1985-364	Lionnet, André	1985		364	basic	Spanish	Uto-Aztecan Languages		Lionnet1985		This concept list is taken from a dataset on Uto-Aztecan languages. The study sets to demonstrate the existence of a Sonorense branch inside the linguistic family and proposes an inner classification of the branch. The data was collected by André Lionnet.	25-57	
 Oliveira-2014-517	Oliveira, Sanderson Castro Soares de	2022		517	basic	English	Panoan languages	https://repositorio.unb.br/handle/10482/17129	Oliveira2014	Oliveira2014	This list presents the concepts that are reconstructed for Proto-Panoan. In the original data, the reflexes are presented for various Panoan languages.	397-491	
-Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
\ No newline at end of file
+Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
+Brown-1837-60	Brown, Nathan	1837		60	basic				Brown1837	Brown1837	An early comparative wordlist designed to enable comparison of the 'Indo-Chinese' languages for scholarly and missionary purposes. The items chosen "are those of the most common objects, and may therefore be regarded the earliest terms in every language, and such as were least liable to be supplanted by foreign words."		
```

Comment:
Maybe English could be add as SOURCE_LANGUAGE and Indo-Chinese languages as TARGET_LANGUAGE. 

Also, pages info could be add: 1023-1038

Reply:
Yes, that should be done!